### PR TITLE
4583 Remove use of status-checker-redirects feature flag

### PR DESCRIPTION
**Pull request by Fbasham**
### Description
per title

### Related Azure Boards Work Items
[AB#4583](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4583)

### Checklist

- [x] I have tested the changes locally
- [ ] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [x] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: gitops/overlays/dev/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/dev/configs/frontend/config.conf:1]: This should be in a separate PR.

File: gitops/overlays/int/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/int/configs/frontend/config.conf:1]: This should be in a separate PR.

File: gitops/overlays/perf/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/perf/configs/frontend/config.conf:1]: This should be in a separate PR.

File: gitops/overlays/prod/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/prod/configs/frontend/config.conf:1]: This should be in a separate PR.

Also, updating the prod configuration will call an immediate deployment to production, which you really don't want. This flag should be removed only when the application has been released and is ready for deployment.

File: gitops/overlays/staging/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/staging/configs/frontend/config.conf:1]: This should be in a separate PR.

File: gitops/overlays/training/configs/frontend/config.conf
Comment (gregory-j-baker) [inline gitops/overlays/training/configs/frontend/config.conf:1]: This should be in a separate PR.

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]
The `gitops/` changes should be in a separate PR.

File: gitops/overlays/prod/configs/frontend/config.conf
Comment (Fbasham) [reply gitops/overlays/prod/configs/frontend/config.conf:1]: ack

**Review by gregory-j-baker — APPROVED** [review]
